app/src/main/java/io/github/droidkaigi/confsched2017/view/helper/Navigator.java

```diff
@@ -68,4 +70,21 @@ public void navigateToWebPage(@NonNull String url) {
         intent.launchUrl(activity, Uri.parse(url));
     }
 
+    public void showConfirmDialog(@StringRes int titleResId, @StringRes int messageResId,
```

Comment:
This should be discussed that `dialog` should be managed by `Navigator` class or not.